frontend/src/components/VAudioDetails/VAudioDetails.vue

```diff
@@ -136,6 +136,6 @@ dt {
 }
 
 dd {
-  @apply pt-2 text-sm text-base font-semibold capitalize leading-[130%];
+  @apply pt-2 text-base text-sm font-semibold capitalize leading-[130%];
```

Comment:
Since `text-base` and `text-sm` are contradictory, they shouldn't both apply simultaneously.